sdk/keyvault/azure-keyvault-keys/tests/_test_case.py

```diff
@@ -0,0 +1,11 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from _test_case_base import KeysTestCaseBase
+from _shared.test_case import KeyVaultTestCase
+
+
+class KeysTestCase(KeysTestCaseBase, KeyVaultTestCase):
```

Comment:
Personally, I would just write e.g.
```py
class TestCaseWithTests(KeysTestCaseBase, KeyVaultTestCase)
```
everywhere instead of adding this empty class.